recipes/swepy/meta.yaml

```diff
@@ -0,0 +1,90 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 13 is conviniet
+# when copying and paster from another recipe, but not really needed.
+{% set name = "swepy" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # If getting the source from GitHub remove the line above
+  # uncomment the line below and modify as needed
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 80605b225adf44b185808036fb012e1fce93d4f4bb035147efd2151bfa72d2c7
+  # sha256 is the prefered checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge
+  #  `conda install openssl -c conda-forge``
+
+build:
+  # Uncomment the following line if the package is pure python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/meta.html#building-noarch-packages for more details.
+  noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    # if your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # compilers are named 'c', 'cxx' and 'fortran'.
+  host:
+    - python
+    - pip
+    - cetbtools
```

Comment:
I do not believe that `cetbtools` or `mapboxgl` need to be in the `host` list since they do not appear to be imported directly or due to an import side-effect in the `setup.py`

It also looks like both of those packages are not on conda-forge yet, so they will need to be submitted for this PR to build.

Reply:
@synapticarbors , this is where I stopped yesterday. Does this mean I need to do this same process first with 'cetbtools' and 'mapboxgl'? Thanks for your help!!

Reply:
It's not a problem. I'd say most packages are maintained by someone other than the creator of the library or package. If you think the creator or maintainer of the library would have any interest in being a co-maintainer of the conda-forge recipe, then you can ping them and see if they would like to be included as a courtesy, but it's not necessary.

Reply:
Great! I very much appreciate the help, and i'll get this sorted out.